spf.gemspec

```diff
@@ -38,35 +39,31 @@ Gem::Specification.new do |s|
   ]
   s.homepage = "https://github.com/agaridata/spf-ruby"
   s.licenses = ["none (all rights reserved)"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23.2"
+  s.rubygems_version = "2.4.6"
   s.summary = "Implementation of the Sender Policy Framework"
 
   if s.respond_to? :specification_version then
-    s.specification_version = 3
+    s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
       s.add_runtime_dependency(%q<ruby-ip>, ["~> 0.9.1"])
-      s.add_development_dependency(%q<rspec>, ["~> 2.8.0"])
-      s.add_development_dependency(%q<rdoc>, ["~> 3.12"])
-      s.add_development_dependency(%q<bundler>, ["~> 1.0"])
-      s.add_development_dependency(%q<jeweler>, ["~> 1.8.7"])
-      s.add_development_dependency(%q<ruby-ip>, ["~> 0.9.1"])
+      s.add_development_dependency(%q<rspec>, ["~> 2.9"])
+      s.add_development_dependency(%q<rdoc>, ["~> 3"])
+      s.add_development_dependency(%q<bundler>, ["~> 1.2"])
+      s.add_development_dependency(%q<jeweler>, ["~> 1.8"])
```

Comment:
We could keep the current versions to support 2.1. ruby-ip was removed since it was added redundantly (it is a runtime dependency not development one, and shouldn't be added twice regardless). Upgrading to rspec 2.9 matches what we've done with other projects, but since we don't really have tests doesn't really have an impact here. The rdoc and jewler dependencies are actually less specific that prior. Are you suggesting lowering them further? Bundler was upgraded without any real thought, so we could drop that back.